src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -385,6 +385,40 @@ public void StartupHookSupport_is_false_by_default_on_trimmed_apps(string target
             }
         }
 
+        [Theory]
+        [InlineData("net5.0")]
+        [InlineData("net6.0")]
+        public void CustomResourceTypesSupport_is_false_by_default_on_trimmed_apps(string targetFramework)
```

Comment:
Can you also delete the warnings in the `ILLink_verify_analysis_warnings_hello_world_app` test above?